packages/button/src/Button.tsx

```diff
@@ -40,6 +41,7 @@ function Button(props: ButtonProps, ref?: ForwardedRef<HTMLButtonElement>) {
       $color={color}
       $square={square}
       $loading={loading}
+      active={active}
```

Comment:
why? It need in modal button. Some UI libs for react or vue have active state of button. 

ButtonStyle dont use the active prop because not in design.